docs/source/development/add-ons/create-an-add-on-prerelease.md

```diff
@@ -0,0 +1,188 @@
+---
+myst:
+  html_meta:
+    "description": "How to create a frontend add-on (development or pre-release)"
+    "property=og:description": "How to create a frontend add-on (development or pre-release)"
+    "property=og:title": "How to create a frontend add-on (development or pre-release)"
+    "keywords": "add-on, Volto, create"
+---
+
+# How to create a frontend add-on (development or pre-release)
+
+This chapter describes how you can create a Volto add-on using the latest **development release** version of Plone with **Volto 18 or later** for the frontend, while having full control over its development and deployment.
+
+```{versionadded} Volto 18.0.0-alpha.43
+{term}`Cookieplone` is now the method to create a Plone add-on with unstable versions of Volto, version 18.0.0-alpha.43 and above.
+```
+
+Follow the steps required to install Cookieplone as described in {doc}`plone:install/create-project-cookieplone`.
+
```

Comment:
Can you verify the system requirements? I don't think we need all of those that are at https://6.docs.plone.org/install/create-project-cookieplone.html#system-requirements, based on the sanity check in the cookieplone template.

Reply:
If you're developing only a frontend add-on, do you need all the other requirements for a backend add-on? Or maybe we just say require them all to keep it simple?